resources/lib/metadata.py

```diff
@@ -222,11 +222,11 @@ def get_tvshowtitle(api_data):
 
         # VRT NU Search API
         if api_data.get('type') == 'episode':
-            return api_data.get('program', '???')
+            return api_data.get('program', '')
 
         # VRT NU Suggest API
         if api_data.get('type') == 'program':
-            return api_data.get('title', '???')
+            return api_data.get('title', '')
```

Comment:
This was done intentionally, so that in case of incorrect metadata, entries would show something and it gives a point of reference to discuss and fix the cause.

Granted, it may not be the best placeholder, we could come up with something better, but an empty string is IMO a step back.

Reply:
It's just that I don't like triple question marks in Kodi GUI.

Reply:
So, you think we need a placeholder in the GUI?

Reply:
Oh I agree that triple question marks is not the best choice, but at least it is identifiable (on screen and in code).

Yes, if we ever get missing keywords where we do not expect them, we would like people to report this, rather than silently ignore. We can only do that by making this visually apparent. The problem by replacing it with "No description available", or "No title available" is that it might be seen as the actual content.

Reply:
Okay, I'll remove this again.